Gemfile

```diff
@@ -46,6 +46,9 @@ gem 'active_model_serializers', '~> 0.10.0'
 # Pagination
 gem 'wor-paginate'
 
+# email
```

Comment:
I'll be picky for a second: sidekiq is not only for async emails, it's for async processes in general (you could use it to process large files without the user noticing it, etc)